portfolio/src/main/webapp/script.js

```diff
@@ -45,13 +45,15 @@ async function getCommentFromFetch() {
     const response = await fetch("/data");
     const comments = await response.json();
     const commentContainer = document.getElementById("comment-container");
+    const maxComments = parseInt(document.getElementById("comment-limit").value, 10);
+    console.log(maxComments);
     commentContainer.innerHTML = "";
-    comments.forEach((commentEntry) => {
-        var commentString = commentEntry.comment;
+    for (var index = 0; index < maxComments; index++) {
+        var commentString = comments[index].comment;
```

Comment:
I'm seeing an error here in my console. My datastore has two comments, FWIW.

```
script.js:52 Uncaught (in promise) TypeError: Cannot read property 'comment' of undefined
    at getCommentFromFetch (script.js:52)
```

Reply:
I forgot to consider the situation that the set limit is larger than the number of comments. So it's not a good idea to set the loop guard to be index < maxComments. Instead, I think I will change the loop guard to index < comments.length and break the loop while the index reaches maxComments.